_episodes/02-working-with-environments.md

```diff
@@ -347,145 +347,6 @@ Running `ls` (linux) / `dir` (Windows) on your anaconda `envs/` directory will l
 > previously installed Conda environments.
 {: .callout}
 
-
```

Comment:
Agree with removing this, its an introductory course so I think its appropriate to keep it simple and have one central place where virtual environments are created and stored, its like teaching people to run before they can walk recommending project specific environments.

Not wanting to denigrate people's experience/knowledge but based on previous experience there is always a proportion who don't understand paths and directory structure as well as might be required and this section has potential to confuse.